### PR TITLE
Update Dockerfile to match current Morpheus Dockerfile

### PR DESCRIPTION
**Pull request by efajardo-nv**
## Description
Docker container build was failing. Dockerfile was using `gpuci/miniforge-cuda` as base image. Morpheus moved to `nvidia/cuda` in 23.03. This PR adds same updates to `morpheus-experimental` Dockerfile.

## Checklist
[x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
[x] New or existing tests cover these changes.
[x] The documentation is up to date with these changes.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -70,13 +102,14 @@ FROM base as runtime
...
+    CONDA_ALWAYS_YES=true /opt/conda/bin/mamba create -n mor_exp -c conda-forge  \
+      python=${PYTHON_VER} && \
     # Clean and activate
     # conda clean -afy && \
-    sed -i 's/conda activate base/conda activate mor_exp/g' ~/.bashrc
+    conda init bash && \
+    echo "conda activate mor_exp" >> ~/.bashrc
 
-# Set the permenant conda channels to use for morpheus experimental
+# Set the permenant conda channes to use for morpheus
```
Comment (mdemoret-nv) [inline docker/Dockerfile:112]:
```suggestion
# Set the permenant conda channels to use for morpheus experimental
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -17,50 +17,82 @@
...
+        libcurand-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
+        libcusolver-${CUDA_MAJOR_VER}-${CUDA_MINOR_VER} \
+        libnuma1 \
+        openjdk-11-jre-headless \
+        openssh-client \
+        pkg-config \
+        tar \
+        unzip \
+        wget \
+        zip \
```
Comment (mdemoret-nv) [inline docker/Dockerfile:73]: Do we need all of these dependencies? This will add significant size to the image

**Review by mdemoret-nv — APPROVED** [review]
Couple of questions to help keep the size low for the container

File: docker/Dockerfile (hunk 2 above)
Comment (efajardo-nv) [reply docker/Dockerfile:73]: no we don't. copy pasted from morpheus dockerfile. i'll remove unused dependencies and retest. thanks.

**Pull request merged**